src/adx.ts

```diff
@@ -74,4 +74,50 @@ export class ADX {
 
         return { adx: this.wema.nextValue(100 * (diAbs / (pDI + nDI))), pdi: pDI, mdi: nDI };
     }
+
+    momentValue(h: number, l: number, c: number) {
+        if (!this.prevClose) {
+            this.prevHigh = h;
```

Comment:
Moment value calculation shouldn't affect indicator state, please dont override prevHigh/prevLow/prevClose values, these values updates only inside of nextValue (when candle is closed)